tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -112,6 +114,26 @@ public function before()
         Context::getContext()->controller = $adminControllerTestDouble;
     }
 
+    /**
+     * @Given I change for the order :orderReference the currency :oldCurrency to :newCurrency
+     *
+     * @param string $orderReference
+     * @param string $oldCurrency
+     * @param string $newCurrency
+     */
+    public function changeOrderCurrency(string $orderReference, string $oldCurrency, string $newCurrency): void
+    {
+        $oldCurrencyId = Currency::getIdByIsoCode($oldCurrency, 0, true);
```

Comment:
```suggestion
        $oldCurrencyId = $this->referenceToId($oldCurrencyReference);
```
Currencies' IDs should be retrievable via the shared storage, if you can use this method instead it would be better for the consistency of the context classes